.pre-commit-config.yaml

```diff
@@ -26,6 +26,10 @@ repos:
     args: [--autofix]
   - id: pretty-format-yaml
     args: [--autofix]
+    # For the labeler GitHub Action, labels with spaces in them must
+    # be put in quotes. However, the pretty-format-yaml hook will
+    # remove the quotes which will break that action, so we should not
+    # run this hook on `labeler.yml`.
```

Comment:
A general rule that I have is that if I spent ≥30 minutes trying to figure something out (or multiple hours, sigh...), it's worth putting in a descriptive comment.